WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsDetailTableViewController.swift

```diff
@@ -0,0 +1,142 @@
+import UIKit
+
+class SiteStatsDetailTableViewController: UITableViewController, StoryboardLoadable {
+
+    // MARK: - StoryboardLoadable Protocol
+
+    static var defaultStoryboardName = "SiteStatsDetailTableViewController"
+
+    // MARK: - Properties
+
+    private typealias Style = WPStyleGuide.Stats
+    private var statSection: StatSection?
+    private weak var siteStatsInsightsDelegate: SiteStatsInsightsDelegate?
+    private weak var siteStatsPeriodDelegate: SiteStatsPeriodDelegate?
+
+    private lazy var tableHandler: ImmuTableViewHandler = {
+        return ImmuTableViewHandler(takeOver: self)
+    }()
+
+    // MARK: - View
+
+    override func viewDidLoad() {
```

Comment:
Perhaps we could delete this method until we need to add some additional behavior?

Reply:
Good catch! It shall be gone!